cockatrice/src/gameselector.cpp

```diff
@@ -74,7 +74,11 @@ GameSelector::GameSelector(AbstractClient *_client,
     connect(filterButton, SIGNAL(clicked()), this, SLOT(actSetFilter()));
     clearFilterButton = new QPushButton;
     clearFilterButton->setIcon(QPixmap("theme:icons/clearsearch"));
-    clearFilterButton->setEnabled(true);
+    if (showFilters && gameListProxyModel->areFilterParametersSetToDefaults()) {
```

Comment:
I feel like @ebbit1q might say to just make this
```suggestion
    clearFilterButton->setEnabled(!showFilters || !gameListProxyModel->areFilterParametersSetToDefaults())
```

Reply:
```suggestion
bool defaultFilters = showFilters && gameListProxyModel->areFilterParametersSetToDefaults()
clearFilterButton->setclearFilterButton->setEnabled(!defaultFilters)
```

Reply:
Done, pushing branch shortly.